app-libs/stf/src/helpers.rs

```diff
@@ -80,12 +80,26 @@ pub fn account_key_hash(account: &AccountId) -> Vec<u8> {
 }
 
 pub fn get_account_info(who: &AccountId) -> Option<AccountInfo> {
-	get_storage_map("System", "Account", who, &StorageHasher::Blake2_128Concat)
+	let maybe_storage_map =
+		get_storage_map("System", "Account", who, &StorageHasher::Blake2_128Concat);
+	if maybe_storage_map.is_none() {
+		warn!("Failed to get account info for account {}", account_id_to_string(who));
```

Comment:
Why `warn`? In case the acc does not have any balance, that's expected behaviour, is it not?

Reply:
yes, true, we could change that to `info` as well